opentech/settings/base.py

```diff
@@ -674,3 +674,35 @@
 PROJECTS_ENABLED = False
 if env.get('PROJECTS_ENABLED', 'false').lower().strip() == 'true':
     PROJECTS_ENABLED = True
+
+
+# Debug Toolbar
+# https://django-debug-toolbar.readthedocs.io/en/latest/index.html
+if DEBUG:
+    INSTALLED_APPS.append('debug_toolbar')
+    MIDDLEWARE = [
+        'debug_toolbar.middleware.DebugToolbarMiddleware',
+        *MIDDLEWARE,
+    ]
+
+INTERNAL_IPS = ['127.0.0.1', '10.0.2.2']
```

Comment:
Should this not be moved in to "if DEBUG:"?

And setting an arbitrary privat ip as 10.0.2.2 on production feels wrong. It's specifically for the VirtualBox environment I believe.

Reply:
That's a [private address](https://en.wikipedia.org/wiki/Private_network#Private_IPv4_addresses) but you're right I'll put it under `DEBUG` just in case.